common/interfaces/desktop_app_interface/usb_event.c

```diff
@@ -161,12 +169,13 @@ static void clear_msg_context() {
 
 static core_error_type_t get_core_req_type(usb_core_msg_t msg,
                                            size_t *request_type) {
-  core_msg_t core_msg_p = CORE_MSG_INIT_ZERO;
+  memzero(&core_msg_p, sizeof(core_msg_t));
   pb_istream_t stream = pb_istream_from_buffer(msg.buffer, msg.size);
   core_error_type_t status = CORE_INVALID_MSG;
   // invalid buffer ref, 0 size & decode failure are error situation
   if (false == pb_decode(&stream, CORE_MSG_FIELDS, &core_msg_p) ||
       NULL == msg.buffer || 0 == msg.size) {
+    // TODO: ADD PBERROR LOG
```

Comment:
```suggestion
        LOG_ERROR("Decoding failed: %s\n", PB_GET_ERROR(&stream));
```

Reply:
This should be covered as separate project wide change

Reply:
yes, we could create a ticket regarding this and add in comments for reference.